pynestml/codegeneration/resources_nest/point_neuron/setup/@MODULE_NAME@.h.jinja2

```diff
@@ -82,7 +82,7 @@ public:
   /**
    * Return the name of your model.
    */
-  const std::string name( void ) const;
+  const std::string name() const;
```

Comment:
The `void` keyword must also be removed from the function in the .cpp file.

Reply:
I am confused.. on the _modernize_ branch of @clinssen I can't see the void keyword being present there..

Reply:
See here: https://github.com/clinssen/nestml/blob/c879b09c7531357904c7dd1bc7a2f63d6a5272b9/pynestml/codegeneration/resources_nest/point_neuron/setup/%40MODULE_NAME%40.cpp.jinja2#L116

Reply:
Ah nevermind, I just didn't read your comment properly. Overlooked the part where you said "in the .cpp file", just read "the void keyword must be removed", looked at the file where you left the comment and was confused. But true, I will create a PR against Charl's branch with the changes I suggested below and also add this suggestion.